src/Libraries/DSOffice/Excel.cs

```diff
@@ -838,7 +862,7 @@ public static object[][] ImportExcel(FileInfo file, string sheetName, bool readA
         /// <param name="writeAsString">Toggle to switch between writing Excel file as strings.</param>
         /// <returns name="data">Data written to the spreadsheet.</returns>
         /// <search>office,excel,spreadsheet</search>
-        public static object[][] ExportExcel(string filePath, string sheetName, int startRow, int startCol, object[][] data, bool overWrite = false, bool writeAsString = false)
+        public static object[][] ExportToExcel(string filePath, string sheetName, int startRow, int startCol, object[][] data, bool overWrite = false, bool writeAsString = false)
```

Comment:
@SHKnudsen how about naming it `ExportExcelAsString`?

Reply:
didnt want to do that as its just an option to export as a string, `ExportExcelAsString` sound like it will export as string by default to me?

Reply:
Agree, but it still says something about the node - that it at least makes that option (to write as string) available. It looks less confusing than `ExportExcel` and `ExportToExcel` IMO.

Reply:
@aparajit-pratap updated the name of the node.

Reply:
@SHKnudsen I'm so sorry, I just realized - the old node `ExportExcel` will be obsoleted and not appear in the library, in which case, I totally agree with you. It makes sense to have the new name, `ExportToExcel` since there will be just this node going forwards. Can we change that back? Really sorry.

Reply:
No worries, I reverted the change so its back to `ExportToExcel`.